DAILY.md

```diff
@@ -17,8 +17,8 @@ days, if you feel like sharing.
 * _(optional)_ Mention any blockers.
 
 ### Daily Examples (taken from our best devs at hipchat):
-  |   |   |
----|---|---|---
+Dev Name | Daily | Time
+---|---|---
 **Ignacio Mendizabal** | Daily: scale testing of conflux and announce i'm gonna be father. | ``11:03 AM``
 **Iñaki Garay** | daily: finishing choosy image processing robustification and starting an epic bughunting rampage | ``11:33 AM``
```

Comment:
I found 6 misspelled words in this sentence: 
- |
	- *Suggested Replacements:* , k, p, u, e, z, j, o, t, d, y, i, n, s, c, x, h, m, r, b, w, g, l, q, a, v, f
- bughunting
- robustification
- |
	- *Suggested Replacements:* , k, p, u, e, z, j, o, t, d, y, i, n, s, c, x, h, m, r, b, w, g, l, q, a, v, f
- Garay**
- **Iñaki